loader/src/sources/albertsons/index.js

```diff
@@ -692,7 +530,7 @@ function formatLocation(data, validAt, checkedAt) {
   return {
     name,
     external_ids,
-    provider: "albertsons",
+    provider: storeBrand.provider_id,
```

Comment:
I think this is what this was always supposed to be, so I went ahead and "fixed" it. (Even though this scraper is deprecated now.)